resources/views/users/profile_review.blade.php

```diff
@@ -16,6 +16,7 @@
 
 #toggle_text {
   cursor: pointer;
+  color: #E7DA3D;
 }
 
```

Comment:
I think we should move this to css file.

Reply:
moved to style.css! thank you.